harness/tests/experiment/fixtures/pytorch_xor_model.py

```diff
@@ -296,6 +297,9 @@ def __init__(self) -> None:
         self.validation_steps_started = 0
```

Comment:
What about adding another PyTorchCallback with the legacy signature for on_training_epoch_end, to make sure the backwards compatibility is not broken?

Reply:
Wait, _this callback_ is using the legacy signature for `on_training_epoch_start` (without the epoch argument). `on_training_epoch_end` is a net new method and therefore has no legacy signature :)

Would you like me to add another callback that uses _the new signature_ ?